_layouts/presentation.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+{% capture reveal_home %}{{ site.baseurl }}/assets/vendor/reveal{% endcapture %}
+{% assign theme = page.theme %}
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>{{ page.title }}</title>
+
+    <meta name="description" content=" {{ page.description }} "/>
+    <meta name="author" content="{{ page.author }}">
+
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
+
+    <link rel="stylesheet" href="{{ reveal_home }}/reveal.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/assets/{{ theme }}" id="theme"/>
+
+    <!-- Printing and PDF exports -->
+    <script type="text/javascript">
+        var link = document.createElement( 'link' );
+        link.rel = 'stylesheet';
+        link.type = 'text/css';
+        link.href = window.location.search.match( /print-pdf/gi ) ? '{{ site.baseurl }}/assets/pdf.css' : '{{ site.baseurl }}/assets/paper.css';
+        document.getElementsByTagName( 'head' )[0].appendChild( link );
+    </script>
+
+    <!--[if lt IE 9]>
```

Comment:
Is this block something that's parsed or can we just delete it?

Reply:
Deleted it.  That was a leftover.  I adapted this layout from another one I did but upgraded reveal.js for this commit.

Reply:
word